### ISSUE TITLE
3.3.1+  Java issue for WINDOWS users

### ISSUE THREAD

**Issue by pir8radio** [signum-network/signum-node#589]
Windows natively only uses JRE  (Java Run-Time Environment).   Windows will never use JDK  (Java Development Kit) without the user manually installing it.      When trying to run on windows you will get this from the Signum EXE.

![image](https://user-images.githubusercontent.com/16748696/152265870-ba0ca958-cad5-4b0c-93e7-447d0579a0c6.png)

Then windows will take you to the Java JRE download site,  but there is no JRE for version 11.   So if you update java on the page windows takes you to, it will do no good.. still JRE 8.     

![image](https://user-images.githubusercontent.com/16748696/152266006-098f3d60-d0ba-4379-bca2-edc6ad52cc31.png)
![image](https://user-images.githubusercontent.com/16748696/152266017-de886998-0c96-453c-aa99-b1ea0fcc2ce2.png)

Come to find out there is no JRE 11,  sounds like in 11 java goes modular, and developer must include packages they wish to use?  Maybe I'm reading this wrong..    https://stackoverflow.com/questions/53733312/where-is-jre-11

Long story short, I just tried to download JDK 11, and of course oracle wont let you download old archived JDK's without having an account...    I didn't want to setup an account, so I downloaded JDK 17, installed and BRS node fired right up..    FYI as a temp workaround.

**Comment by frankTheTank72** [comment]
We will add https://www.techspot.com/downloads/5553-java-jdk.html in the readme - here you can download Java11 for Windows

**Comment by frankTheTank72** [comment]
@pir8radio i think we can close that or ? As we have it now already included with Java

**Comment by jjos2372** [comment]
Java is now included withing the windows package.
